.changeset/chatty-crews-visit.md

```diff
@@ -0,0 +1,6 @@
+---
+"@khanacademy/wonder-blocks-form": major
```

Comment:
(As we discussed on Slack) This could be a `minor` update since changing a prop from being required to optional won't be a breaking change to existing callers. It could also remain a `major` update if we want to emphasize there's a change to the external API!